src/course/views.py

```diff
@@ -87,6 +89,25 @@ def post(self):
         print course_info
         return json.dumps(course_info)
 
+class ArchiveCourse(MethodView):
```

Comment:
Dont worry about changing this, but for future endeavors, it would probably make sense to make this a more generic function that takes in a boolean value for whether to archive or unarchive